index.js

```diff
@@ -88,7 +106,10 @@ console.log('hello world')
  * Example Output: "I called an arrow function!"
  */
 
-// TODO: ADD CODE HERE
+const myArrowFunction = () => {
+    return "I am exited about programming and coding!!!";
+};
```

Comment:
Great work Eugene!
This is totally correct, but if you wanted to this can also be written more concisely (don't think you'd lose any clarity) as
`const myArrowFunction = () => "This is a function expression!";`
^ You're allowed to write it like that when there's nothing else you need to do in the code block beside return. It's sort of a shorthand.